tycho-compiler-plugin/src/main/java/org/eclipse/tycho/compiler/AbstractOsgiCompilerMojo.java

```diff
@@ -738,7 +742,7 @@ protected CompilerConfiguration getCompilerConfiguration(List<String> compileSou
                 if (jreClasspathEntry.isModule()) {
                     Collection<String> modules = jreClasspathEntry.getLimitModules();
                     if (!modules.isEmpty()) {
-                        compilerConfiguration.addCompilerCustomArgument("--limit-modules", String.join(",", modules));
+                        addCompilerCustomArgument(compilerConfiguration, "--limit-modules", String.join(",", modules));
```

Comment:
This is a standard javac option, too

Reply:
It seems not to be a standard plexus compiler option so I'm not sure how to handle this gracefully...